### PR TITLE
DOCS-243 New integration guide for developing e-signature solutions

### PR DESCRIPTION
**Pull request by jimj-procoretech**
New integration guide requested by Partnerships team covering e-signature solutions. See https://procoretech.atlassian.net/browse/DOCS-243 for background information.

### CODE REVIEW

File: tutorials/tutorial_e_signature_integration.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+## Recommended Approach
+
+For technology partners interested in integrating an electronic signature solution with Procore, we recommend leveraging the Procore API in combination with our embedded app experience.
+(See our [Embedded App Helper](https://developers.procore.com/documentation/building-apps-helper-library) for information on implementing an embedded experience iframe within Procore.)
+
+An integration can pull PDF documents from Procore to be signed in an external e-signature application.
+Signed copies of the document can be returned to Procore’s Documents tool (at a minimum), allowing users to save executed documents within Procore as a project's system of record.
+Permissions around this tool can also add an extra layer of security to keep needed documents confidential. 
+Additional functionality can be added to the integration for completing business processes.
+For example, Procore’s API can be utilized to also update the status of a contract to “Complete,” record the date the document was signed, and mark the contract as executed.
```
Comment (claudioprocore) [inline tutorials/tutorial_e_signature_integration.md:18]: Both here…

File: tutorials/tutorial_e_signature_integration.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+## Documents Tool Integration
+
+Additional information on integrating with the Documents tool can be found at [developers.procore.com](https://developers.procore.com/documentation/tutorial-uploads).
+We recommended that executed documents be saved to a folder within the Procore Documents tool for reference.
+This tool allows for Admin-level users to make folders private. We also recommend this approach for all contract documents.
+
+## Additional Functionality
+
+Additional functionality can be added to the integration to support contract execution.
+Through Procore’s API, an integration can update the status of a contract to “Complete,” mark the contract as executed and record the executed date.
```
Comment (claudioprocore) [inline tutorials/tutorial_e_signature_integration.md:86]: …and here, is `"Complete,"` correct, or should it be `"Complete",` ?

File: tutorials/tutorial_e_signature_integration.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+This might add some complexity to the update as it requires a different API call.
+(See information below.)
+
+Here are links to the API documentation covering the update actions discussed above for contracts, change orders, and workflows in Procore:
+
+### Contracts
+
+- [Prime Contract](https://developers.procore.com/reference/rest/v1/prime-contracts#update-prime-contract)
+- [Commitments](https://developers.procore.com/reference/rest/v1/work-order-contracts#update-work-order-contract) (Subcontracts/Work Order)
+- [Commitments](https://developers.procore.com/reference/rest/v1/purchase-order-contracts#update-purchase-order-contract) (Purchase Order)
```
Comment (claudioprocore) [inline tutorials/tutorial_e_signature_integration.md:99]: The last two bullet points both have the text "Commitments". However, the links point to different resources (Work order contracts and Purchase order contracts). Is the text correct?

File: tutorials/tutorial_e_signature_integration.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+
+### Change Orders
+
+- [General Change Order Information](https://developers.procore.com/documentation/tutorial-change-orders)
+- [Change Order](https://developers.procore.com/reference/rest/v1/change-order-packages#update-change-order-package)
+(Note: Best practice would be to use the APIs to pull an “Approved Change Order” into the technology partner software through webhooks.)
```
Comment (claudioprocore) [inline tutorials/tutorial_e_signature_integration.md:105]: Adding a new line after a bullet point in markdown breaks out of the list. Should this note be on the same line as the bullet point above?

Similar comment for the bullet point below.

File: tutorials/tutorial_e_signature_integration.md [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+**Note:** Procore allows *Custom Workflows* to be assigned with these tools.
+Custom Workflow in this scenario mostly refers to internal “controls” or “approvals” prior to something being sent out.
+This might add some complexity to the update as it requires a different API call.
+(See information below.)
+
+Here are links to the API documentation covering the update actions discussed above for contracts, change orders, and workflows in Procore:
+
+### Contracts
+
+- [Prime Contract](https://developers.procore.com/reference/rest/v1/prime-contracts#update-prime-contract)
```
Comment (claudioprocore) [inline tutorials/tutorial_e_signature_integration.md:97]: Is this URL the correct one to publish in the guide? I opened it and it seems like it's marked as "alpha":

Snap 2021-07-08 at 1 21 45 PM

**Review by claudioprocore — COMMENTED** [review]
Thanks @jimj-procoretech! Left a few inline comments.

File: tutorials/tutorial_e_signature_integration.md (hunk 3 above)
Comment (jimj-procoretech) [reply tutorials/tutorial_e_signature_integration.md:99]: As specified by the Partnerships team, yes, this is the correct text. 'Commitments' in this context refers to the tool (category), not the resource.

File: tutorials/tutorial_e_signature_integration.md (hunk 5 above)
Comment (jimj-procoretech) [reply tutorials/tutorial_e_signature_integration.md:97]: You are probably logged in as a Procore employee? The link is correct. External users will be taken to the v1.0 version of the resource which is in production.

**Review by claudioprocore — APPROVED** [review]

**Pull request merged**